internal/lefthook/install.go

```diff
@@ -99,7 +99,7 @@ func (l *Lefthook) configExists(path string) bool {
 }
 
 func (l *Lefthook) createConfig(path string) error {
-	file := filepath.Join(path, config.DefaultConfigName)
+	file := filepath.Join(path, config.DefaultDotConfigName)
```

Comment:
Let's keep this behavior unchanged? We'll change this in later major release. I think it would be great to just support `.lefthook.yml` and `.lefthook-local.yml` by now, fix possible issue, decide on what to do when both variants exist and so on. So, this transition will be smooth. WDYT?

Reply:
That sounds good. The docs don't mention of the dot-prefixed variants so there is no change of confusion if one is generated or has priority over the non-dot-prefixed variants.